.azure-pipelines/pipelines.yml

```diff
@@ -474,6 +474,11 @@ stages:
         AZP_BRANCH: $(Build.SourceBranch)
         NETLIFY_TRIGGER_URL: $(NetlifyTriggerURL)
 
+    - script: bazel run //tools/project:publish
```

Comment:
not sure if we can/should use bazel directly here

also this needs a github token with permission to create a tag/release